src/main/java/org/alliancegenome/curation_api/base/BaseService.java

```diff
@@ -38,6 +38,11 @@ public E delete(String id) {
         return dao.remove(id);
     }
 
+    @Transactional
+    public E delete(Long id) {
+        return dao.remove(id);
+    }
+
```

Comment:
@cmpich good catch I didn't update these after we made the GeneratedCurie base class